src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,17 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        userService.findByEmail(email);
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        if (user.getEmail().equals(email) && user.getPassword().equals(password)) {
+            return true;
+        } else {
+            return false;
+        }
+
```

Comment:
remove this block of code and write this line instead of it:
`return user != null && user.getPassword().equals(password);`